<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgAnimation</class>
 <widget class="QWidget" name="WdgAnimation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>82</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="playbackLayout">
     <item>
      <widget class="QToolButton" name="btnPreviousFrame">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnNextFrame">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="keyframingLayout">
     <item>
      <widget class="QToolButton" name="btnAddKeyframe">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblInfo">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
